### PR TITLE
Initial cleanup

### PR DESCRIPTION
**Pull request by robertaboukhalil**
This PR is to:

-  Remove box for pasting file contents (SAM/coords)
-  Fix panels that can't be collapsed
-  Remove unused code
-  Remove unused JS libraries in js/
-  Use modern JS APIs for: dict_length(), download()

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
+    "dev": "python3 -m http.server 1234 || python -m SimpleHTTPServer 1234"
   },
   "author": "Maria Nattestad",
   "license": "MIT",
   "dependencies": {
     "@biowasm/aioli": "^3.0.0",
     "bootstrap": "3.4.1",
     "d3": "3.5.6",
-    "jquery": "3.5.0",
+    "jquery": "3.5.1",
```
Comment (robertaboukhalil) [inline package.json:19]: A bug in 3.5.0 caused the `collapsible` divs to stop working

File: js/index_ribbon.js [hunk 2]
```diff
@@ -6859,11 +6712,8 @@ function submit_bam_url() {
 }
 d3.select("#submit_bam_url").on("click", submit_bam_url);
 
-window.addEventListener("beforeunload", function (e) {
-  var confirmationMessage = "Leave Ribbon?";
-
-  (e || window.event).returnValue = confirmationMessage; //Gecko + IE
-  return confirmationMessage; //Gecko + Webkit, Safari, Chrome etc.
+window.addEventListener("beforeunload", function (event) {
+  event.preventDefault();
```
Comment (robertaboukhalil) [inline js/index_ribbon.js:6716]: Confirmation messages not supported in modern browsers

File: js/cross-browser-split.js
Comment (robertaboukhalil) [inline js/cross-browser-split.js:1]: This library overwrites String.split(), not needed for modern browsers

File: js/jquery-ui.min.js
Comment (robertaboukhalil) [inline js/jquery-ui.min.js:1]: Already bundled in package.json

**Review by MariaNattestad — APPROVED** [review]
Nice modernizations! I love that a few imports like cross-browser-split are just not needed anymore because modern browsers have standardized to support those tasks.

**Pull request merged**